pair-programming/week-3/form-validation/script.js

```diff
@@ -1,20 +1,8 @@
-// In your JS file, write a function that changes the value of the variable `testBoolean` each time the button is clicked.
-// Notice that I'm already giving you the connector `"onclick="toggle()"` inside the HTML of the button, so you don't need to worry about it.
-
-// ## Bonus
-// What do you need to do to change the name of the function?
-
-let testBoolean = true;
-console.log("Default value of your boolean is " + testBoolean);
-
-function toggleValue() {
-  if (testBoolean) {
-    testBoolean = false;
-
-    console.log("The value of your boolean is " + testBoolean);
-  } else {
-    testBoolean = true;
-
-    console.log("The value of your boolean is " + testBoolean);
-  }
-}
+// filepath: week-3/form-validation/script.js
+// Add your form validation logic here.
+//
+// The validation rules are as follow:
+// - The two inputted passwords must be identical.
+// - The minimal password length is 6 characters.
+// - The password must contain at least one digit.
+// - The validation result must be shown on the page with an appropriate message in each case.
```

Comment:
This code is guidance to get them started to fulfilling the prompt. Let's make update as follows:

```suggestion
// Write a function that changes the value of the variable `testBoolean` each time the button is clicked.
// Notice that you have been given the `toggle()` which will need to be used inside the HTML of the button for your `onclick` event

// Consider the following to get started:
// What do you need to do to change the name of the function?

// Remember your validation rules are as follows:
// - The two inputted passwords must be identical.
// - The minimal password length is 6 characters.
// - The password must contain at least one digit.
// - The validation result must be shown on the page with an appropriate message in each case.

let testBoolean = true;
console.log("Default value of your boolean is " + testBoolean);

function toggleValue() {
  if (testBoolean) {
    testBoolean = false;

    console.log("The value of your boolean is " + testBoolean);
  } else {
    testBoolean = true;

    console.log("The value of your boolean is " + testBoolean);
  }
}
```